rewrite-gradle/src/main/java/org/openrewrite/gradle/UpgradeDependencyVersion.java

```diff
@@ -68,13 +68,15 @@ public class UpgradeDependencyVersion extends Recipe {
     String artifactId;
 
     @Option(displayName = "New version",
-            description = "An exact version number or node-style semver selector used to select the version number.",
+            description = "An exact version number or node-style semver selector used to select the version number. " +
+                          "You can also use `latest.release` for the latest available version and `latest.patch` if " +
+                          "the current version is a valid semantic version.",
```

Comment:
I like this, but I think it would be even better if we could also add a Markdown link to the documentation page on [version selectors](https://docs.openrewrite.org/reference/dependency-version-selectors). On that page I notice that the documentation for `latest.patch` is missing.